op_crates/fetch/lib.rs

```diff
@@ -185,7 +187,7 @@ where
     request = request.header(name, v);
   }
 
-  let fut = request.send();
+  let fut = tokio::spawn(async move { request.send().await });
```

Comment:
What's the reason for this change?

Reply:
My findings that that there was a subtle race condition here I was able to surface from this change. This is a race condition that I think is not actually present in the single threaded runtime that could only be introduced in the multithreaded one, but I may be incorrect and this existing in the current runtime. When calling send, we kick off an async operation, however we don't await on it until we call the later function, which I believe is opFetchSend. If the connection resolves between opFetch and opFetchSend, the runtime may not be read to receive the connection and it causes and error. To prevent this, I push the task onto a worker where it will be await'd until we call opFetchSend.

Reply:
I think this changes the behavior from "suspended until polled in `op_fetch_send()`" to "runs concurrently". I don't know this code well enough to say if that's good, bad, or neutral, but @bartlomieju or @lucacasonato probably do.

Aside: this can be simplified to `let fut = tokio::spawn(request.send());`